packages/application-generic/package.json

```
@@ -72,6 +72,7 @@
     "@novu/netcore": "^0.19.0",
     "@novu/nodemailer": "^0.19.0",
     "@novu/one-signal": "^0.19.0",
+    "@novu/pushpad": "^0.16.3",
```

```suggestion
    "@novu/pushpad": "^0.20.0-alpha.1",
```
After fixing the merge conflicts, update this version to `^0.20.0-alpha.1`

---

@collimarco , I think this change was removed after the rebase, can you update this and then run npm run setup:project to generate the new pnpm-lock file